src/Java/gtPlusPlus/core/material/Material.java

```diff
@@ -37,6 +37,7 @@
 	public static final Set<Material> mMaterialMap = new HashSet<Material>();
 
 	public static final Map<String, Map<String, ItemStack>> mComponentMap = new HashMap<String, Map<String, ItemStack>>();
+	public static final Map<String, ItemStack> mOreDictItemCache = new HashMap<String, ItemStack>();
```

Comment:
No.. You've still yet to elaborate why you're doing this.
We don't need to add a cache for OD names when you can just cache the result from OD itself.

Reply:
wut?

Reply:
```
		String aKey = aPrefix.name();
		Map<String, ItemStack> g =  mComponentMap.get(this.unlocalizedName);
		g.put(aKey, u);
		mComponentMap.put(unlocalizedName, g);
```
You're not putting the results in the material map for the given prefix.